example-dockingandviewports/src/ofApp.h

```diff
@@ -11,11 +11,16 @@ class ofApp : public ofBaseApp{
 
 		void setup() {
             // Setup imgui with the appropriate config flags
-            gui.setup(nullptr, false, ImGuiConfigFlags_DockingEnable | ImGuiConfigFlags_ViewportsEnable, true);
+            ofxImGui::BaseTheme* theme_ = nullptr;
+            bool autoDraw_ = false; 
+            ImGuiConfigFlags customFlags_ = ImGuiConfigFlags_DockingEnable | ImGuiConfigFlags_ViewportsEnable;
+            bool restoreGuiState_ = true;
+            bool showImGuiMouseCursor_ = false;
+            gui.setup(theme_, autoDraw_, customFlags_, restoreGuiState_, showImGuiMouseCursor_);
 
             // Uncomment below to perform docking with SHIFT key
             // Gives a better user experience, matter of opinion.
-            //ImGui::GetIO().ConfigDockingWithShift=true;
```

Comment:
Nice, makes the example clearer. Could you just undo the comment of this line before merging ?

Reply:
Ok. 
But FYI, if "shift key docking" is not enabled, docking does not seem to work here in Windows... 
This is normal maybe? What am I missing?